### PR TITLE
Reduce test screen sizes

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
In an effort to reduce our Travis CI build times and prevent timeouts, the following screen sizes have been removed from various tests.

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const timeZones = ['America/Chicago', 'America/Denver', 'America/Detroit', 'Amer
   'America/Los_Angeles', 'America/New_York', 'America/Phoenix'];
 const includesTZ = timeZones.includes(moment.tz.guess());
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```
Comment (mhemesath) [inline packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js]: Is there a need for 3 breakpoitns here? Wondering if we can just do tiny and large.

File: packages/terra-date-picker/tests/nightwatch/date-picker-spec.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```
Comment (mhemesath) [inline packages/terra-date-picker/tests/nightwatch/date-picker-spec.js]: Is there a need for 3 breakpoitns here? Wondering if we can just do tiny and large.

File: packages/terra-dynamic-grid/tests/nightwatch/dynamic-grid-spec.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```
Comment (mhemesath) [inline packages/terra-dynamic-grid/tests/nightwatch/dynamic-grid-spec.js]: we probably only need tiny/large here.

File: packages/terra-embedded-content-consumer/tests/nightwatch/embedded-content-consumer-spec.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```
Comment (mhemesath) [inline packages/terra-embedded-content-consumer/tests/nightwatch/embedded-content-consumer-spec.js]: this probably only needs one breakpoint. All it does is mount XFC and doesn't need any responsive tests.

**Comment by bjankord** [comment]
@mhemesath Updated test breakpoints: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be

File: packages/terra-dynamic-grid/tests/nightwatch/dynamic-grid-spec.js (hunk 3 above)
Comment (bjankord) [reply packages/terra-dynamic-grid/tests/nightwatch/dynamic-grid-spec.js]: Updated: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be

File: packages/terra-date-picker/tests/nightwatch/date-picker-spec.js (hunk 2 above)
Comment (bjankord) [reply packages/terra-date-picker/tests/nightwatch/date-picker-spec.js]: Updated: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be

File: packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js (hunk 1 above)
Comment (bjankord) [reply packages/terra-date-time-picker/tests/nightwatch/date-time-picker-spec.js]: Updated: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be

File: packages/terra-embedded-content-consumer/tests/nightwatch/embedded-content-consumer-spec.js (hunk 4 above)
Comment (bjankord) [reply packages/terra-embedded-content-consumer/tests/nightwatch/embedded-content-consumer-spec.js]: Updated: https://github.com/cerner/terra-core/pull/1166/commits/3f3d3d93af165fad40df984045b5a2425b87fc8b

**Comment by mhemesath** [comment]
Excellent time savings! I think we can shave time off our wdio tests by using saucelabs to run multiple instances at once.. or setting up AWS to run browser tests.

**Pull request merged**